imperial_coldfront_plugin/views.py

```diff
@@ -585,6 +586,17 @@ def add_rdf_storage_allocation(request):
                 value=dart_id,
             )
 
+            gid_attribute_type = AllocationAttributeType.objects.get(name="GID")
+            gid = get_new_gid()
+            AllocationAttribute.objects.create(
+                allocation_attribute_type=gid_attribute_type,
+                allocation=rdf_allocation,
+                value=gid,
+            )
+            AllocationAttributeUsage.objects.create(
+                allocation_attribute=gid_attribute_type, value=0
+            )
+
```

Comment:
Don't forget to pass the gid value below when `_ldap_create_group` is added to the chain below.